src/components/Settings/SettingsMain.tsx

```diff
@@ -181,7 +181,7 @@ const SettingsMain: React.FC = () => {
                         <input
                           type="text"
                           id="apiKey"
-                          className="rounded-l-only"
+                          className="cursor-not-allowed rounded-l-only"
```

Comment:
This field can be interacted with, though. You can click and drag to select inside the box and copy manually if that's your jam. Having the "not allowed" cursor makes this more difficult.

Reply:
Ah.  Perhaps we shouldn't modify the API key field, in that case.  I still think we should change the cursor for the Plex server name though, since that one is kinda confusing and I don't see why you'd want to copy that?

Reply:
Yes. I only commented about the API key field after all :)